alphastats/dataset/dataset.py

```diff
@@ -161,6 +150,37 @@ def _check_loader(loader):
                 "Invalid index_column: consider reloading your data with: AlphaPeptLoader, MaxQuantLoader, DIANNLoader, FragPipeLoader, SpectronautLoader"
             )
 
+    def _create_id_dicts(self, sep: str = ";") -> Tuple[dict, dict, dict]:
+        """Create mapprings from gene, protein to feature and from feature to repr."""
+
+        features = self.mat.columns.to_list()
+        gene_to_features_map = defaultdict(lambda: [])
+        protein_to_features_map = defaultdict(lambda: [])
+        feature_to_repr_map = defaultdict(lambda x: x)
+
+        for proteins, feature in self.rawinput[[Cols.INDEX, Cols.INDEX]].itertuples(
```

Comment:
`[Cols.INDEX, Cols.INDEX]` typo?

Reply:
Nope, that's intended. One is split, the other is left as is.

Reply:
I see this could be without any zip, but see it as future proofing if we ever enable other feature ids (which would make proteins and features yield from different columns)

Reply:
I would prefer having easy to understand code now over proofing it for a future that could come or not :-)